qiita_pet/templates/study_base.html

```diff
@@ -15,12 +16,26 @@
 
   function fill_artifact(aid) {
     $("#study-main").html("ARTIFACT " + aid + " HTML STUFF");
+    $.get("/admin/artifact/", {artifact_id: +aid })
```

Comment:
In which case will not an integer be passed here? If this code is only used in our HTML and no-body else can call this function, I don't think it is needed to have this check. Which type of error is returned through the interface in case that this is not an integer? Is this just to check if a programmer messed up?